changelogs/current.yaml

```diff
@@ -39,6 +39,56 @@ new_features:
 
 - area: http
   change: |
+    added the expected :ref:`receive <envoy_v3_api_field_config.core.v3.HealthCheck.HttpHealthCheck.receive>` payload check for HTTP health check.
```

Comment:
there's a lot of unrelated changes in this version history which might comes from resolving conflicts, please merge main again and add only what you did in this PR.

Reply:
OK, fixed, thanks!